<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmYScrollBar</class>
 <widget class="QWidget" name="frmYScrollBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="YScrollBarH" name="hscrollbarButton" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>21</width>
     <height>371</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="YScrollBarV" name="vscrollbarButton" native="true">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>31</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>YScrollBarV</class>
   <extends>QWidget</extends>
   <header location="global">yscrollbarv.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>YScrollBarH</class>
   <extends>QWidget</extends>
   <header location="global">yscrollbarh.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
